File: 2ndDay/1.py
skills = []
max = None
index = [0, 0]
for i in range(4):
    skills.append(input().split(" "))
    for j in range(3):
        if max == None:
            max = int(skills[i][j])
            index = [i, j]
        elif int(skills[i][j]) > max:
            max = int(skills[i][j])
            index = [i, j]
print(index[0] + 1)

File: 2ndDay/1.rb
skills = Array.new()
max = nil
index = 0
for i in 0..3
    skills.append(gets.split(" ").map(&:to_i))
    for j in 0..2
      if max == nil
        max = skills[i][j]
        index = i + 1
      elsif skills[i][j] > max
        max = skills[i][j]
        index = i + 1
      end
    end
end
puts index